site/content/blog/2025-02-15-portswigger-lab-file-path-traversal-simple-case/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Solving Portswigger Lab File Path Traversal Simple Case with ZAP"
+summary: "Video and explanation of How to Solve the Portswigger labs using ZAP, in this case: 'Path Traversal Simple Case'"
+date: 2025-02-15
+tags:
+- blog
+draft: true
+authors:
+    - Bash Bunny
+---
+
+This [Portswigger Lab](https://portswigger.net/web-security/file-path-traversal/lab-simple) has a path traversal vulnerability, which allows anyone to retrieve any file from the system the webpage is hosted in. In this particular case the laboratory consist on retrieving the `/etc/passwd` file, demonstrating this way the vulnerability.
+
+If you wan to know more about Path Traversal, I suggest looking at the [ZAP Path Traversal](https://www.zaproxy.org/docs/alerts/6-2/) website.
```

Comment:
"wan" -> "want"
"https://www.zaproxy.org/docs/alerts/6-2/" -> "/docs/alerts/6-2/" (all links should be relative)
"website" -> "alert page"